### ISSUE TITLE
BL7 upgrade: use browser default color for focus indicator

### ISSUE THREAD

**Issue by ggeisler** [sul-dlss/exhibits#1625]
Some, but not all, page elements now show the border-color that indicates focus as red, instead of the browser default (blue).

The offending elements seem to be form inputs and buttons that use the `btn-primary` styling. Other buttons and form elements seem to still have the default blue color. 

All the focus indicators should be the same color, ideally just the default browser blue that users are used to seeing.

<img width="1073" alt="Screen Shot 2020-02-05 at 4 54 34 PM" src="https://user-images.githubusercontent.com/101482/73894017-71a18280-4838-11ea-974a-8b376ae7a845.png">
